lib/onigumo.ex

```diff
@@ -32,6 +31,13 @@ defmodule Onigumo do
     File.write!(file_path, body)
   end
 
+  def load_urls(path \\ nil)
+
```

Comment:
Hmm I am confused what does it mean. Do you have some material or docs where did you find this approach, please?